portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/Properties.testcase

```diff
@@ -375,6 +375,139 @@ definition {
 		}
 	}
 
+	@description = "Bug ID: LRAC-8093 | Automation ID: LRAC-11533 | Test Summary: Able to clear property data"
+	@priority = "4"
+	test ClearPropertyData {
+		property analytics.cloud.upstream = "false";
```

Comment:
I think this test could be run in the upstream routine. Did you notice any problems in this routine?

Reply:
Oh yea probably not needed. I was testing Known Individuals but changed it to Total Individuals so probably should work in upstream as well